src/components/scenes/ChangePasswordScene.tsx

```diff
@@ -121,34 +122,34 @@ const ChangePasswordSceneComponent = ({
           </EdgeAnim>
         )}
         <EdgeAnim enter={{ type: 'fadeInUp', distance: 25 }}>
-          <OutlinedTextInput
-            value={password}
-            secureTextEntry
-            returnKeyType="next"
-            label={lstrings.password}
-            autoFocus={focusFirst}
-            onChangeText={handleValidatePassword}
-            onSubmitEditing={handleFocusSwitch}
-            clearIcon
-            searchIcon={false}
-            marginRem={[0, 0.75, 1.25]}
-            maxLength={100}
-          />
+          <Space horizontal={0.75} bottom={1.25}>
```

Comment:
Due to now known performance impact of additional `View` components, I would push back against the architecture of the `Space` component and to use padding on existing components instead where possible.

Reply:
What known performance impact of additional view components?